lib/pages/news_feed/view/news_feed_page.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:acs_upb_mobile/pages/news_feed/model/news_feed_item.dart';
+import 'package:acs_upb_mobile/pages/news_feed/service/news_feed_provider.dart';
+import 'package:acs_upb_mobile/resources/utils.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class NewsFeedPage extends StatelessWidget {
+  static const String routeName = '/news_feed';
+
+  @override
+  Widget build(BuildContext context) {
+    final newsFeedProvider = Provider.of<NewsFeedProvider>(context);
+
+    return AppScaffold(
+      title: S.of(context).newsFeedTitle,
+      body: FutureBuilder(
+        future:
+            newsFeedProvider.fetchNewsFeedItems(context: context, limit: null),

```

Comment:
In Dart, you can simply omit named parameters instead of passing `null`:
```suggestion
            newsFeedProvider.fetchNewsFeedItems(context: context),

```